keepsorted/options_test.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keepsorted
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+func TestBlockOptions(t *testing.T) {
```

Comment:
Huh, have I been using mercurial for so long that I've forgotten Git doesn't have manual copy flagging? I was gonna suggest we mark this file as copied from the existing one so that it's easy to see what changed in this and the other tests you moved, but it seems git only has this for moved files? Bah, whatever. I'm assuming you didn't change these tests.

Reply:
Yeah... I just moved the tests to a new file to start breaking up the 2000 line keep_sorted_test.go file